### PR TITLE
1st draft for panel dashboard

### PR DESCRIPTION
**Pull request by apkrelling**
Quite a few things still need to be done here; but as of now it is functional - I just need to do minor fixes to display the images.

### CODE REVIEW

**Comment by apkrelling** [comment]
In order to have the images (widgets, map, plot, etc) displayed, I set up the environment.yml so that hvplot and holoviews would be installed from the pyviz channel, and not conda-forge. I'm not sure if this is the ideal setup - @ocefpaf , any ideas?

File: ERDDAP_timeseries_explorer.ipynb [hunk 1]
```diff
@@ -15,7 +15,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from erddap_app.layout import layout"
```
Comment (ocefpaf) [inline ERDDAP_timeseries_explorer.ipynb:18]: The header of this notebook has some outdated test. Do you mind removing it in this PR? This notebook no longer uses any tech from Jason Grout's talk.

**Comment by ocefpaf** [comment]
The very first code executed is a `progressbar` that will get updated when we run the next cell. It is odd to see it on top of its trigger. Can we get the progressbar triggered as an output of `get_valid_stdnames`?

File: environment.yml [hunk 2]
```diff
@@ -1,11 +1,13 @@
 name: widgets
 channels:
   - conda-forge
+  - pyviz
```
Comment (ocefpaf) [inline environment.yml]:
```suggestion
```

File: environment.yml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
   - python=3.8
   - pandas
   - pendulum
-  - ipyleaflet
-  - bqplot
   - erddapy
   - ipykernel
+  - bokeh
+  - pyviz::holoviews
+  - pyviz::hvplot
```
Comment (ocefpaf) [inline environment.yml]:
```suggestion
  - holoviews
  - hvplot
```

These changes worked on my computer. All plots showed as expected.

File: environment.yml [hunk 4]
```diff
@@ -1,11 +1,13 @@
 name: widgets
 channels:
   - conda-forge
+  - pyviz
 dependencies:
   - python=3.8
```
Comment (ocefpaf) [inline environment.yml]:
```suggestion
  - python=3.9
```

File: ERDDAP_timeseries_explorer.ipynb (hunk 1 above)
Comment (apkrelling) [reply ERDDAP_timeseries_explorer.ipynb:18]: I removed the outdated information from the header. However, if you prefer, I can just take out the entire header.

Comment (ocefpaf) [reply ERDDAP_timeseries_explorer.ipynb:18]: Let's leave that for later. We'll need a new one.

**Pull request merged**